intrahospital_api/loader.py

```
@@ -367,40 +130,31 @@ def async_load_patient(patient_id, patient_load_id):
         raise
 
 
+@timing
+def load_demographics(hospital_number):
+    started = timezone.now()
+    api = get_api()
+    try:
+        result = api.demographics(hospital_number)
+    except:
```

Why the bare except here?

---

To stop it blowing up in the pathway.